apps/openstmap/interface.html

```diff
@@ -183,7 +183,7 @@
                     }).addTo(map);
                     let marker = new L.marker(latlon).addTo(map);
                     map.fitBounds(latlon.toBounds(2000/*meters*/), {animation: false});
-                  }, 100);
+                  }, 500);
```

Comment:
osm interface was always loading for me with
the previous 100ms timeout, no idea why.

Since there already was a timeout I assume
that I was not the first with this problem..

Reply:
That's odd - but glad this fixes it for you!